CMakeLists.txt

```diff
@@ -90,6 +113,7 @@ include_directories( include )
 include_directories( CanLibLoader/include )
 include_directories( CanInterface/include )
 
+
 if(NOT STANDALONE_BUILD)
 	add_library ( CanModule OBJECT ${SRCS} )
 	set( SERVER_LINK_LIBRARIES ${SERVER_LINK_LIBRARIES} ${CMAKE_DL_LIBS} PARENT_SCOPE )
```

Comment:
I believe you shouldn't control SERVER_LINK_LIBRARIES from a child module, rather the parent (here: quasar CMakeLists) should find a way to tell what to link with.

Reply:
sorry, I prefer not to touch this now because the "NOT STANDALONE_BUILD" is quite unclear to me. Also just more libs are added as CanModule needs them, that seems right. The parent can not know that...